packages/runtime/datastore-definitions/src/channel.ts

```diff
@@ -21,13 +21,19 @@ export interface IChannel extends IFluidLoadable {
 
     /**
      * Generates summary of the channel synchronously.
+     * @param fullTree - flag indicating whether the attempt should generate a full
```

Comment:
How important is this "attempt"?

Is it a bug or just a perf issue if the implementation returns a summary tree contining a handle for an unchanged subtree? What about a handle for something other than an unchaed subtree (es: a new blob)?

Its important for the documenration on this method to make it possible to tell if an imlementation of it is incorrect, and without answering the above, there are cases where I wouldn't know.

Reply:
The attach summary cannot contain summary handles, i.e., path to sub-trees (or blobs) in the previous summary. It can however contain ISummaryAttachment, i.e., handles to blobs that were uploaded async via the blob manager.
Since attach summary is generated for local channels when making them visible to remote clients, they don't have any previous summaries to compare against. So, they cannot really send summary handles.

I agree with you that its worth calling it out here.